routes/studentRoutes.js

```diff
@@ -4,7 +4,15 @@ var Student = require(path.join(__dirname, '../models/student'));
 
 routes.GETallStudents = function(req, res) {
   Student.find({}, function(err, allStudents) {
```

Comment:
okay so I might be crazy, but I never actually see where these extra variables get used. I feel like im missing something, but I've looked around and I can't find them